Doc/library/codecs.rst

```diff
@@ -841,8 +841,8 @@ The design is such that one can use the factory functions returned by the
    code calling :meth:`read` and :meth:`write`, while *Reader* and *Writer*
    work on the backend — the data in *stream*.
 
-   You can use these objects to do transparent transcodings from e.g. Latin-1
-   to UTF-8 and back.
+   You can use these objects to do transparent transcodings, e.g., from Latin-1
+	to UTF-8 and back.
```

Comment:
This is failing the lint check for tabs.

Reply:
Oops, thanks!